File: 05217.py
for i in range(int(input())):
    n = int(input())

    print(f"Pairs for {n}:", end=' ')
    if n % 2:
        if n == 1:
            print()
        else:
            print(1, n - 1, sep=' ', end='')
            for j in range(2, n // 2 + 1):
                print(',', j, n - j, sep=' ', end='')
            print()
    else:
        if n == 2:
            print()
        else:
            print(1, n - 1, sep=' ', end='')
            for j in range(2, n // 2):
                print(',', j, n - j, sep=' ', end='')
            print()

File: 05217.cpp
#include <iostream>
using namespace std;

int main() {
	int t; cin >> t;
	for (int i = 0; i < t; i++) {
		int n; cin >> n;

		cout << "Pairs for " << n << ": ";
		if (n % 2) {
			if (n == 1) cout << endl;
			else {
				cout << 1 << ' ' << n - 1;
				for (int i = 2; i <= n / 2; i++)
					cout << ", " << i << ' ' << n - i;
				cout << endl;
			}
		}
		else {
			if (n == 2) cout << endl;
			else {
				cout << 1 << ' ' << n - 1;
				for (int i = 2; i <= n / 2 - 1; i++)
					cout << ", " << i << ' ' << n - i;
				cout << endl;
			}
		}
	}
}
